torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -258,8 +259,46 @@ struct FlattenInputs {
     end_dim = 2,
   };
 };
+
+// Indexes of aten::topk inputs.
+struct TopKInputs {
+  enum {
+    input = 0,
+    k = 1,
+    dim = 2,
+    largest = 3,
+    sorted = 4,
+  };
+};
 } // namespace
 
+caffe2::TypeMeta PyTorchModelLoader::convertGlowType(glow::TypeRef ty) {
+  switch (ty->getElementType()) {
+  case ElemKind::FloatTy:
+    return caffe2::TypeMeta::Make<float>();
+  case ElemKind::Float16Ty:
+    return caffe2::TypeMeta::Make<int16_t>(); // TODO
+  case ElemKind::Int8QTy:
+    return caffe2::TypeMeta::Make<int8_t>();
+  case ElemKind::UInt8QTy:
+    return caffe2::TypeMeta::Make<uint8_t>();
+  case ElemKind::Int16QTy:
+    return caffe2::TypeMeta::Make<int16_t>();
+  case ElemKind::Int32QTy:
```

Comment:
let's leave the qtypes out of it because they should probably map to different qtypes in pytorch in the future